src/styles.scss

```diff
@@ -562,4 +562,20 @@ body .ui-tabview.ui-tabview-top .ui-tabview-nav li.ui-state-active, body .ui-tab
 
 .ui-table-scrollable-body {
     max-height: 665px;
+}
+
+body {
```

Comment:
What was this styling for in the accordion?

Reply:
yes. In order to get the settings menus to not display behind different views in when tiling different views, I appended the dialog boxes to body and therefore had to update the css.